src/core/syscalls/other_syscalls.rs

```diff
@@ -0,0 +1,72 @@
+use crate::core::errors::syscall_handler_errors::SyscallHandlerError;
+use cairo_rs::{
+    hint_processor::builtin_hint_processor::{
+        builtin_hint_processor_definition::HintProcessorData,
+        hint_utils::{get_integer_from_var_name, insert_value_from_var_name},
+    },
+    vm::{errors::hint_errors::HintError, vm_core::VirtualMachine},
+};
+use felt::Felt;
+use num_traits::{One, Zero};
+use std::{collections::HashMap, ops::Shl};
+
+pub fn addr_bound_prime(
+    vm: &mut VirtualMachine,
+    hint_data: &HintProcessorData,
+    constants: &HashMap<String, Felt>,
+) -> Result<(), SyscallHandlerError> {
+    let addr_bound = constants
+        .get("starkware.starknet.common.storage.ADDR_BOUND")
+        .ok_or(HintError::MissingConstant(
+            "starkware.starknet.common.storage.ADDR_BOUND",
+        ))?;
+
+    let lower_bound = Felt::from(1).shl(250u32);
+    let upper_bound = Felt::from(1).shl(251u32);
+    if !(&lower_bound < addr_bound && addr_bound <= &upper_bound) {
+        return Err(HintError::AssertionFailed(
+            "normalize_address() cannot be used with the current constants.".to_string(),
+        )
+        .into());
+    }
+
+    let addr = get_integer_from_var_name("addr", vm, &hint_data.ids_data, &hint_data.ap_tracking)?;
+    let is_small = if addr.as_ref() < addr_bound {
+        Felt::one()
+    } else {
+        Felt::zero()
+    };
+
+    insert_value_from_var_name(
+        "is_small",
+        is_small,
+        vm,
+        &hint_data.ids_data,
+        &hint_data.ap_tracking,
+    )?;
+
+    Ok(())
+}
+
+pub fn addr_is_250(
+    vm: &mut VirtualMachine,
+    hint_data: &HintProcessorData,
+    _constants: &HashMap<String, Felt>,
```

Comment:
We can remove this argument it it isn't used

Reply:
Fixed.